geomstats/spd_matrices_space.py

```diff
@@ -153,17 +217,35 @@ def exp(self, tangent_vec, base_point):
 
         This gives a symmetric positive definite matrix.
         """
-        sqrt_base_point = scipy.linalg.sqrtm(base_point)
+        if tangent_vec.ndim == 2:
```

Comment:
Seems that you’re using this patter quite often in your code. Want to refactor this?

Reply:
Good point, thank you. I will add a TODO for now.